languageServer/src/server.ts

```diff
@@ -0,0 +1,118 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
```

Comment:
[Q] Which extensions/docs did you use to come up with this?

Reply:
https://github.com/microsoft/vscode-extension-samples/tree/main/lsp-sample

I missed these, sorry. In my defence It's hard to notice comments in most highlight schemes.
I installed this https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments

It seems to be working now